pyanaconda/core/regexes.py

```diff
@@ -194,3 +194,6 @@
 
 # Octet of MAC address
 MAC_OCTET = re.compile(r'[a-fA-F0-9][a-fA-F0-9]')
+
+# Name of initramfs connection created by NM based on MAC
+NM_MAC_INITRAMFS_CONNECTION = re.compile(r'^([0-9A-Fa-f]{2}:){5}([0-9A-Fa-f]{2})$')
```

Comment:
It would be good to have add some unit tests for this regex, that can be done in upstream.